docs/getting-started/generated-app.md

```diff
@@ -0,0 +1,53 @@
+---
+id: generated-app
+title: The generated app
+sidebar_label: The Generated App
+slug: /generated-app
+---
+
+# The Generated App
+
+Applications created with Amplication are generated as human-readable and editable TypeScript code.
+
+In this article you will find the details on the structure of the application and the technologies behind it.
+
+## Technologies
+
+The generated app is built with the following amazing open-source technologies:
+
+### Server-side
+
+- [NestJS](https://nestjs.com/) - A progressive Node.js framework for building efficient, reliable and scalable server-side applications.
+- [Prisma](https://www.prisma.io/) - Next-generation ORM
+  for Node.js and TypeScript.
+- [PostgreSQL](https://www.postgresql.org/) - The World's Most Advanced Open Source Relational Database
+- [Passport](http://www.passportjs.org/) - Simple, unobtrusive authentication for Node.js
+- [GraphQL](https://graphql.org/) - a query language for APIs.
+- [Swagger UI](https://swagger.io/) - Visual documentation for REST API based on OpenAPI Specification.
+- [Jest](https://jestjs.io/) - delightful JavaScript Testing Framework with a focus on simplicity.
```

Comment:
@yuval-hazaz same type of comment: remove the marketing fluff ('delightful')

Reply:
In this section, we use the marketing phrase of the packages as it is. I think it is acceptable as this is the minimum "payment" to these open-source project we use in out project.
I understand you want to keep the docs "dry" but when talking about other technologies and not about ours I think it is acceptable.

@iddan as an open-source long-timer, when reading the documentation is it batters you?